docusaurus/docs/recipes/video-player.md

```diff
@@ -0,0 +1,99 @@
+---
+id: video-player
+title: Adding a video player
+sidebar_label: Video player
+---
+
+It's often normal to have the necessity of having videos in your website. In this recipe we will show you how to use [`react-player`](https://github.com/CookPete/react-player) and some common use cases.
+
+## Walk-through
+
+### 1. Install `react-player`
+
+```sh
+npm i react-player
+```
+
+### 2. Use `ReactPlayer` in your project
+
+You just need to import `ReactPlayer` and start using it.
+
+```js
+// ...
+import ReactPlayer from 'react-player';
+
+const Home = () => (
+    <div className={ styles.home }>
+        <ReactPlayer url="https://www.video-url.com" />
+    </div>
+);
+```
+
+### 3. Usage
```

Comment:
Not sure this is a good title for use cases, `usage` is a broad term that does not specify any particular user need. For example, consider how `Usage` is interchangeable with the `Walk-through` title in this document. I would maybe name this `Common use cases` to be as specific to this section as possible.